pkg/network/protocols/http/etw_interface.go

```diff
@@ -24,17 +29,48 @@ type EtwInterface struct {
 	captureHTTP        bool
 	captureHTTPS       bool
 	requestSize        int64
+
+	// ETW component
+	httpguid windows.GUID
+	session  etw.Session
 }
 
-//nolint:revive // TODO(WKIT) Fix revive linter
-func NewEtwInterface(c *config.Config) *EtwInterface {
-	return &EtwInterface{
+// NewEtwInterface returns a new EtwInterface instance
+func NewEtwInterface(c *config.Config) (*EtwInterface, error) {
+	ei := &EtwInterface{
 		maxEntriesBuffered: c.MaxHTTPStatsBuffered,
 		DataChannel:        make(chan []WinHttpTransaction),
 		captureHTTPS:       c.EnableNativeTLSMonitoring,
 		captureHTTP:        c.EnableHTTPMonitoring,
 		requestSize:        c.HTTPMaxRequestFragment,
 	}
+	etwSessionName := "SystemProbeUSM_ETW"
```

Comment:
💭: not a blocking comment for this PR, however, do you think that the `etwSessionName` would make sense to add to the `EtwInterface` and then pass it in as part of the `Config` for future users of the `NewEtwInterface` function? Just thinking about what would happen if 2 places called this with the same `etwSessionName`.